lib/ansible/modules/system/mount.py

```diff
@@ -76,7 +66,7 @@
         point will be created.
       - Similarly, specifying C(absent) will remove the mount point directory.
     required: true
-    choices: ["present", "absent", "mounted", "unmounted"]
+    choices: [ absent, mounted, present, unmounted ]
```

Comment:
There should be no spaces at the beginning and the end of the square brackets content.

Reply:
This is YAML, for Python you're right because of PEP8.

Reply:
I couldn't find any better example:

https://en.wikipedia.org/wiki/YAML#Basic_components